app/models/summaryReturnsModels.scala

```diff
@@ -90,6 +90,8 @@ case class SummaryReturnsModel(atedBalance: Option[BigDecimal] = None,
                               )
 
 object SummaryReturnsModel {
+  val newSocialHousingDescription = "Provider of social housing or housing co-operative"
```

Comment:
Should this be hardcoded text in the Model or utilise the message file?

Reply:
It's being checked against an english text entry so for now I think I can be hardcoded (or moved to config)